internal/store/projects.go

```diff
@@ -128,7 +129,8 @@ func (s *Store) CreateProject(name, orgID, regionUsageRestrictions string, defau
 func (s *Store) DeleteProject(projectID string) error {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		_, err := s.client.(*atlas.Client).Projects.Delete(s.ctx, projectID)
+		res, err := s.clientv2.ProjectsApi.DeleteProject(s.ctx, projectID).Execute()
+		defer res.Body.Close()
 		return err
```

Comment:
Side note. Error information in OpenAPI file doesn't include JSON based information. 

```
	var openapiError v1client.GenericOpenAPIError

	if ok := errors.As(err, &openapiError); ok {
		errModel := openapiError.Model()

		thisIsAnParsedAPIError, ok := errModel.(v1client.Error)
		if !ok {
			return nil
		}
		fmt.Print(thisIsAnParsedAPIError)
		```

Reply:
Without this change: "Api error occurred" 
After casting "Cannot create project. Id is missing

Reply:
I'm happy to include that method into sdk, but it is good to keep that first in the CLI.